pennylane/wires.py

```diff
@@ -173,6 +176,14 @@ def tolist(self):
         """
         return list(self.labels)
 
+    def toset(self):
```

Comment:
good to cache this?

Reply:
we mutates sets sometimes so it may not be the best idea for debuggability.